pages/main/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { OrbitControls } from '@react-three/drei'
+import { Canvas } from '@react-three/fiber'
+import styled from '@emotion/styled'
+import dynamic from 'next/dynamic'
+
+import CubeLoader from '@/components/common/Loader/CubeLoader'
+import CustomSuspense from '@/components/common/Suspense/CustomSuspense'
+
+const MainRoom = dynamic(() => import('../../components/main/MainRoom'))
+
+export default function Main() {
+  return (
+    <>
+      <CustomSuspense
+        fallback={
+          <LoaderContainer>
+            <CubeLoader />
+          </LoaderContainer>
+        }
+        maxDuration={5500}
+      >
+        <Canvas camera={{ position: [1, 1.7, 1] }} style={{ height: '100vh' }}>
+          <directionalLight position={[7, 7, 10]} />
+          <MainRoom position={[0, 0.75, 0]} scale={[0.22, 0.22, 0.22]} />
+          <OrbitControls target={[0, 1.4, 0]} />
+        </Canvas>
+      </CustomSuspense>
+    </>
```

Comment:
하위노드가 전부 CustomSuspense 안에 들어가있는데 fragment는 빼도 괜찮지 않을까용??
```suggestion
    <CustomSuspense
      fallback={
        <LoaderContainer>
          <CubeLoader />
        </LoaderContainer>
      }
      maxDuration={5500}
    >
      <Canvas camera={{ position: [1, 1.7, 1] }} style={{ height: '100vh' }}>
        <directionalLight position={[7, 7, 10]} />
        <MainRoom position={[0, 0.75, 0]} scale={[0.22, 0.22, 0.22]} />
        <OrbitControls target={[0, 1.4, 0]} />
      </Canvas>
    </CustomSuspense>
```

Reply:
오 그렇네용! 반영하겠슴돠